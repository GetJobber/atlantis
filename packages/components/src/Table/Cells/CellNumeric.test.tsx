import React from "react";
import renderer from "react-test-renderer";
import { CellNumeric } from "../";

it("renders a numeric cell", () => {
  const tree = renderer.create(<CellNumeric value={13} />).toJSON();
  expect(tree).toMatchSnapshot();
});
