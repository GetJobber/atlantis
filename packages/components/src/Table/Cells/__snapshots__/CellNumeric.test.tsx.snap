// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a numeric cell 1`] = `
<td
  className="cell right"
>
  <span
    className="numeric"
  >
    13
  </span>
</td>
`;
